src/99-appendices/04-entity-table.md

```diff
@@ -13,20 +13,22 @@ A general introduction to entities is given in the section on
-| channels<br>(meg/eeg/ieeg)                                                                     | REQUIRED      | OPTIONAL      | REQUIRED       |               |                          |                |                          | OPTIONAL      |                        |                |                     |                       |                 |                 |
-| headshape<br>(meg)                                                                             | REQUIRED      | OPTIONAL      |                | OPTIONAL      |                          |                |                          |               |                        |                |                     |                       | OPTIONAL        |                 |
-| markers<br>(meg)                                                                               | REQUIRED      | OPTIONAL      | OPTIONAL       | OPTIONAL      |                          |                |                          |               |                        |                |                     |                       | OPTIONAL        |                 |
-| photo<br>(meg/eeg/ieeg)                                                                        | REQUIRED      | OPTIONAL      |                | OPTIONAL      |                          |                |                          |               |                        |                |                     |                       |                 |                 |        
-| electrodes<br>(eeg/ieeg)                                                                       | REQUIRED      | OPTIONAL      |                | OPTIONAL      |                          |                |                          |               |                        |                |                     |                       | OPTIONAL        |                 |
-| events<br>(meg/eeg/ieeg)                                                                       | REQUIRED      | OPTIONAL      | REQUIRED       |               |                          |                |                          | OPTIONAL      |                        |                |                     |                       |                 |                 |        
+| anat<br>(T1w T2w T1rho T1map T2map T2star FLAIR FLASH PD PDmap PDT2 inplaneT1 inplaneT2 angio) | REQUIRED      | REQUIRED*     |                | OPTIONAL      | OPTIONAL                 | OPTIONAL       |                          |               |                        |                |                     |                       |                 |                 |
```

Comment:
This is going to conflict with #475. @tsalo Is this something that can be accommodated in the new structure?

I think it might be worth just having a sentence making clear that, although OPTIONAL, if the Session entity has any value, it MUST appear in both the directory structure and file name.

Reply:
Fine with me.  Neither "REQUIRED" nor "OPTIONAL" is really clear in this case.  I also considered putting the asterisk on the column name instead of each cell.  Or if we don't like the asterisk, moving the sentence before the table is fine too.

Reply:
I was under the assumption that "OPTIONAL" generally referred to entities that would, in specific cases, be required. E.g., `echo` is an optional entity, but in cases where there are multiple echoes it is definitely required, so I agree that this should be reflected in the definition. The only one that immediately comes to mind as truly optional in all cases is `acq` (although I would like to perhaps change that in #530).

For #475, I could add a third label for entities easily enough, although I'd prefer to use something without special characters. Perhaps something like "CONDITIONAL"? The actual logic, where conditions under which an entity is required are defined in the specification, would have to wait for a larger conversion to a machine-parseable format.

Reply:
I second "CONDITIONAL" given that rfc2119 defines "OPTIONAL" as being "truly optional".  Not a great definition, but I think we agree on what it means.  

For this PR however, how about I just remove my edits to the table?

Reply:
CONDITIONAL sounds nice to me --> we'd just have to "extend" the rfc2119 for our purposes.

re: this PR: +1 to remove edits to entity table and merge this PR as a small clarification. Could you @Moo-Marc then open a new issue to track the discussion on that some entities which are currently OPTIONAL, are actually CONDITIONAL (our new potential extension of rfc2119)?

Reply:
Done!